java/client/src/main/java/glide/api/models/commands/stream/StreamClaimOptions.java

```diff
@@ -0,0 +1,119 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.stream;
+
+import glide.api.commands.StreamBaseCommands;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+/**
+ * Optional arguments to {@link StreamBaseCommands#xclaim(String, String, String, long, String[],
+ * StreamClaimOptions)}
+ *
+ * @see <a href="https://redis.io/commands/xclaim/">redis.io</a>
+ */
+@Builder
+public class StreamClaimOptions {
+
+    /** Redis api string to designate IDLE time in milliseconds */
+    public static final String IDLE_REDIS_API = "IDLE";
+
+    /** Redis api string to designate TIME time in unix-milliseconds */
+    public static final String TIME_REDIS_API = "TIME";
+
+    /** Redis api string to designate RETRYCOUNT */
+    public static final String RETRY_COUNT_REDIS_API = "RETRYCOUNT";
+
+    /** Redis api string to designate FORCE */
+    public static final String FORCE_REDIS_API = "FORCE";
+
+    /** Redis api string to designate JUSTID */
+    public static final String JUST_ID_REDIS_API = "JUSTID";
+
+    /** Redis api string to designate LASTID */
+    public static final String LAST_ID_REDIS_API = "LASTID";
+
+    /**
+     * Set the idle time (last time it was delivered) of the message. If <code>idle</code> is not
+     * specified, an <code>idle</code> of <code>0</code> is assumed, that is, the time count is reset
+     * because the message has now a new owner trying to process it.
+     */
+    private final Long idle; // in milliseconds
+
+    /**
+     * This is the same as idle but instead of a relative amount of milliseconds, it sets the idle
```

Comment:
as `{@link #idle`

Reply:
Do these 2 time arguments conflict if both set?

Reply:
Do we need such detailed descriptions for them though?

Reply:
I just copied from the ValKey documentation and didn't feel it was necessary to fiddle with it.